README.md

```diff
@@ -4,11 +4,11 @@
 
 ## Integration
 
-### Android
+### Gradle (JVM, Android, KMP)
 
 Add this to your dependencies:
 
-`implementation 'io.stepuplabs.spaydkmp:spayd-kmp-android:<latest-version>'`
+`implementation 'io.stepuplabs.spaydkmp:spayd-kmp:<latest-version>'`
```

Comment:
I'm not sure about changing the artifact name. Is it because the code doesn't have any Android-specific dependencies and it's pure JVM code?

Reply:
Exactly, there is nothing Android-specific right now. The whole Android-specific implementation can be removed, with the removal of minSdk limitation as a positive side effect.

Several generated modules for different platforms should be resolved correctly when using root `spayd-kmp`.
```
spayd-kmp
spayd-kmp-iosarm64
spayd-kmp-iossimulatorarm64
spayd-kmp-iosx64
spayd-kmp-js
spayd-kmp-jvm
spayd-kmp-wasm-js
```

For some reason, it doesn't work this way when using the deployer publishing, but works with `publishToMavenLocal` task and default `shared-*` artifacts.

I will investigate it further.

Reply:
OK, I agree with removing Android specific implementation and changing the artifactId.

We are using [MavenDeployer plugin](https://github.com/deepmedia/MavenDeployer) for deploying to Central. It also support local deploy, so please try to deploy locally and see what artifacts in generated.

When it will be published to Central, there will be just one artifact or multiple?

Also, I'm not sure how it works in Javascript world with the KMP & dependencies - should we deploy to some JS specific repository like NPM?